inventory/host_vars/stratum0host.example

```diff
@@ -0,0 +1 @@
+cvmfs_srv_device: /dev/sdb
```

Comment:
what about adding this to `local_site_specific_vars.yml` ? and then use here something like:
```
cvmfs_srv_device: "{{ local_cvmfs_srv_device | default('/dev/sdb') }}"
```

Reply:
But I think this is machine specific? Maybe a site has a stratum 0 and stratum 1, and for the first they want to use /dev/sdb and for the latter /dev/sdc?

Reply:
ups, I see this is in an example file and will always be a site specific settings so I think it's ok like it is. Please ignore my comment

Reply:
you are totally right. I missed it's an example. IMO it's good to go